driver/bpf/fillers.h

```diff
@@ -1754,9 +1762,12 @@ static __always_inline u32 bpf_map_id_up(struct uid_gid_map *map, u32 id)
 
 	if (extents <= UID_GID_MAP_MAX_BASE_EXTENTS) {
 		extent = bpf_map_id_up_base(extents, map, id);
-	} else {
+	} 
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(4, 15, 0))			
+	else {
```

Comment:
So, this branch will be skipped on old kernel versions? 
I am curious about this one! :) 
Thanks!

Reply:
Yes, it is correct, kernel 4.15 increased the number of extents to `340`, in all the previous kernels the limit is still `5` so the `if` case should be sufficient. You can see the commit corresponding to the increment [here](https://github.com/torvalds/linux/commit/6397fac4915ab3002dc15aae751455da1a852f25).

Reply:
I see, thanks! 
I think a small comment there would be helpful!

Reply:
I have added it, thank you for your suggestion.